mlflow/utils/async_logging/run_operations.py

```diff
@@ -21,25 +16,25 @@ def wait(self):
                 failed_operations.append(e)
 
         if len(failed_operations) > 0:
-            # Importing MlflowException gives circular reference / module load error, need to
-            #  figure out why.
             raise MlflowException(
-                "The following failures occurred while performing one or more logging"
-                + f" operations: {failed_operations}"
+                "The following failures occurred while performing one or more async logging "
+                f"operations: {failed_operations}"
             )
 
 
 def get_combined_run_operations(run_operations_list: [RunOperations]) -> RunOperations:
```

Comment:
`[RunOperations]` is not a valid type annotation.

Reply:
good catch